data/card_catalogue.json

```diff
@@ -573,27 +625,17 @@
         {
           "title": "Magic Trick",
           "upgrade": "",
-          "tier": "blue",
+          "tier": "green",
           "details": {
             "rules": "An aura surrounds you. NPCs who fail a perception check believe this spell had the effect of your choice.",
             "flavour": "\"I don't know exactly what he did.. but I just don't feel.. right? Know what I mean?\" -Barry the Bandit",
             "image": "assets/core/card/illustration/Magic Trick.png"
           }
         },
-        {
-          "title": "Animal Accomplice",
-          "upgrade": "",
-          "tier": "blue",
-          "details": {
-            "rules": "Summons the magical imprint of a loyal wolfhound, which dissapates when it takes or deals damage. Will follow you until commanded to attack a target for 5 melee damage",
-            "flavour": "Her name is Daisy",
-            "image": "assets/core/card/illustration/Animal Accomplice.png"
-          }
-        },
```

Comment:
I think we should give the whole pet thing another go separately, I feel like this iteration is more complicated then its effect is interesting